requirements.txt

```diff
@@ -17,8 +17,8 @@ SQLAlchemy
 behave
 kombu==3.0.35
 pexpect  # From liblinphonelib
-psycopg2==2.6.2
+psycopg2
 python-consul==0.7.0
-pyyaml==3.12
+pyyaml
```

Comment:
Why you need to remove the version? The xivo-libpython is only tested to work with this specific version

Reply:
A leftover of a local change, I will restore it.

Reply:
Are you sure, @bloom1 make a special commit to remove it

Reply:
I don't remember why I needed this, maybe @bloom1 does ?

Reply:
I probably removed it because of it not being compatible with Python 3.7. Usually I would have left it at a local change but @pc-m told me that it would be ok to remove it but I don't remember why exactly.